### ISSUE TITLE
an empty sequence line still passes

### ISSUE THREAD

**Issue by biocyberman** [alastair-droop/fqtools#6]
The second entry of this fastq still passes the validator. Is it something intentional?

```
@LD5V2:07687:11026
CGGGGGTCTTAGCTTTGGCTCTCCTTGCAAAGTTATTTCTAGTTAATTCATTATGCAGAAGGTATAGGGGTTAGTCCTTGCTTATATTATGCTTGGTTATAATTTTTCATCTTTCCCTTGCGGTACTATATCTATTGCGACCA
+
35977*6772999990959:;:<6<.53::19;39891845..*-36159<6;::::;;6;6>:95333)52577957*...*/6774999894726268858=>=-99:99:2;>3>5:::5:;;<;;::;7/,*,3***+,
@LD5V2:07687:11043

+

@LD5V2:07688:11020
AAAATTTAACACCCATAGTAGGCCTAAAAGCAGCCACCAATTAAGAAAGCGTTCAAGCCCAACACCCACTACCTAAAAAATCCCAAACATATAACTGAACT
+
:::0:?4>7==<<4<;;;=<=7;78888.9;@><5;;4:4:4:4:;;2;<<<6<<6<<=4>6;;<=2;;;;;5565533'5::/;<2<?ABD?7<<<;5<=

```

**Comment by alastair-droop** [comment]
Hi.

Yes, the inclusion of empty reads in valid FASTQ files is intentional. Empty reads will often cause downstream issues, so should be checked for and eliminated, but are sometimes necessary, for example if trimming paired files seperately.

Cutadapt can yield zero-length reads, for example:

~~~bash
echo "@TEST
AATGAATAGATAGATATAGATATAGATAGATAGATATAGATAGA
+
AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA" > test.fastq
cutadapt -aAATGAATAGATAGATATAGATATAGATAGATAGATATAGATAGA test.fastq > out.fastq
cat out.fastq
~~~

yields:
~~~
@TEST

+

~~~

To summarise, zero-length reads are problematic, but are indeed valid FASTQ.
